### PR TITLE
Added new params to stream source/sink.

### PR DESCRIPTION
**Pull request by StoyanTc**
Document the new authentication parameters for stream source/sink.
Ref. [CEP-519](https://macrometa.atlassian.net/browse/CEP-519)

[CEP-519]: https://macrometa.atlassian.net/browse/CEP-519?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by diana-macrometa — CHANGES_REQUESTED** [review]
Please also update the 17.12 release notes if necessary. 

release-notes > release-notes-0-17-12

The write-up for this update starts on line 59.

**Comment by StoyanTc** [comment]
> Please also update the 17.12 release notes if necessary.
> 
> release-notes > release-notes-0-17-12
> 
> The write-up for this update starts on line 59.

@diana-macrometa - Where the release-notes are?

**Comment by diana-macrometa** [comment]
In docs > release-notes > release-notes-0-17-12

File: docs/cep/source/stream-source.md [hunk 1]
```diff
@@ -59,6 +63,10 @@ The following parameters are used to configure a stream definition.
 | num.listener.threads | The number of listener threads. | 1 | INT | Yes |
 | num.io.threads | The number of listener threads. | 1 | INT | Yes |
 | receiver.queue.size | The number of messages accumulated by a consumer before an application calls. | 1000 | INT | Yes |
+| stream.url | The URL of the Pulsar broker, e.g. pulsar_ssl://my-broker:6651. | NULL | STRING | Yes |
```
Comment (diana-macrometa) [inline docs/cep/source/stream-source.md:66]: The mention of Pulsar in this and the next line seem out of place. I have no idea how I would find any of the Pulsar things in our setup.

**Review by diana-macrometa — CHANGES_REQUESTED** [review]
I am confused. Why are we talking about Pulsar in discussing Macrometa source and stream? Can users even see the URL of the Pulsar broker and such?

File: docs/cep/source/stream-source.md (hunk 1 above)
Comment (StoyanTc) [reply docs/cep/source/stream-source.md:66]: @diana-macrometa - Here this is valid. We didn't have an ability to access external Pulsar cluster in our stream sources and sinks till now but Intel could be one such customer.

**Review by diana-macrometa — APPROVED** [review]

**Pull request merged**